package/MDAnalysis/coordinates/PDB.py

```diff
@@ -391,14 +391,22 @@ def _read_frame(self, frame):
             elif line[:6] == 'CRYST1':
                 # does an implicit str -> float conversion
                 try:
-                    self.ts._unitcell[:] = [line[6:15], line[15:24],
-                                            line[24:33], line[33:40],
-                                            line[40:47], line[47:54]]
+                    cell_dims = [line[6:15], line[15:24],
```

Comment:
As per my previous comment, you only want the `cell_sim` extraction in the `try` block. The reason for not adding the rest is that you want to limit the amount of operations you do here so that your `try` is specifically encompassing the operation you are expecting to fail.